appcenter-link-scripts/src/ios/PodFile.js

```diff
@@ -107,15 +108,25 @@ Podfile.prototype.isTargetPlatformDefined = function (platform) {
 };
 
 Podfile.prototype.getTargetSectionPattern = function () {
-    const patterns = this.fileContents.match(/# Pods for .*/);
-    if (patterns === null) {
-        throw new Error(`
-    Error: Could not find a "# Pods for" comment in your Podfile. Please add a "# Pods for AppCenter" line
-    in ${this.file}, inside
-    the "target" section, then rerun the react-native link. AppCenter pods will be added below the comment.
-    `);
+    const projectDirectory = path.resolve(__dirname, '..', '..', '..', '..');
```

Comment:
Concat four `'..'` to find the project directory assumes "appcenter-link-scripts" directory is at "/node_modules/appcenter-link-scripts" rather than multiple level down under the "/node_modules" folder like "/node_modules/appcenter/appcenter-link-scripts". This shouldn't be a problem for anyone using npm v3 or later, so it's good. (I tested this pr works with npm v3.10.10, which is the version of npm on App Center Build VMs.)